doc/rst/language/spec/procedures.rst

```diff
@@ -477,8 +477,7 @@ The Inout Intent
 When ``inout`` is specified as the intent, the actual argument is copied
 into the formal argument as with the ``in`` intent and then copied back
 out as with the ``out`` intent. The actual argument must be a valid
-lvalue. The formal argument can be modified within the function. The
-type of the actual argument must be the same as the type of the formal.
+lvalue. The formal argument can be modified within the function.
```

Comment:
In the above:
* "copied back" is a different language than what is used for the out-intent. Can we make it consistent?
* I suggest adding something like "the type of the actual argument must be acceptable to both an in-formal and an out-formal of the same type as the inout formal" or "must satisfy the requirements of" or ...